tests/Fixtures/TestBundle/Entity/UuidIdentifierDummy.php

```diff
@@ -17,6 +17,8 @@
 /**
  * Custom identifier dummy.
  *
+ * @author Exploit.cz <insekticid@exploit.cz>
```

Comment:
lol no I wondered the same

IMO we should keep as `@author` the one who created the class not uppon each edit :). Should be fixed in other classes too. Still, what's the point of adding `@author` when we can `git blame`?